.github/actions/setup_dbt/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Setup dbt
+description: Install dependencies for dbt and configure it for use in CI.
+inputs:
+  role-to-assume:
+    description: AWS IAM role to assume when running dbt operations.
+    required: true
```

Comment:
From an interface perspective, it might make sense to also expose the optional input variables that the nested actions called by this action accept. I decided not to do that here since so far none of our scripts are overriding any of the defaults for those input variables, and I don't yet see a case where it would be useful for them to do so.

Reply:
Also: I think you know this since you've already run into this hurdle before, but for extra clarity, the `role-to-assume` argument is exposed here since composite actions currently do not support reading from repository secrets.